specs/player_spec.rb

```diff
@@ -0,0 +1,97 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+require_relative '../lib/scoring'
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+
+  before do
+    @john = Scrabble::Player.new("John")
+  end
+
+  describe 'initialize' do
+    it 'Takes in a name' do
+      @john.must_respond_to :name
+      @john.name.must_equal "John"
+      @john.name.must_be_kind_of String
+    end
+
+    it 'returns an array' do
+      @john.played_words.must_be_kind_of Array
+      @john.played_words.must_equal []
+    end
+  end
+
+  describe 'play' do
+    it 'takes in a word' do
+      @john.must_respond_to :play
+      @john.play("table").must_be_kind_of Integer
+      @john.play(20).must_be_nil
+    end
+
+    it 'word is added to play array' do
+      @john.play("snail")
+      @john.played_words.must_include "snail"
+    end
+
+    it 'returns false if game is over (player has already won)' do
+      @john.total_score = 100
+      @john.play('dog').must_equal false
+    end
+
+    it 'returns score of word' do
+      @john.play("snail").must_equal 5
+      @john.play("snail").must_be_kind_of Integer
+    end
+  end
+
+  describe 'total_score' do
+    it 'returns sum of played words' do
+      @john.played_words.push("board", "cup")
```

Comment:
Again you're hacking the class.  Go through the public interface and make sure that playing the words affects the list and the sum.